src/foundry/common/data/data.mjs/drawingData.d.ts

```diff
@@ -27,7 +27,7 @@ interface DrawingDataSchema extends DocumentSchema {
   rotation: FieldReturnType<typeof fields.ANGLE_FIELD, { default: 0 }>;
   z: typeof fields.REQUIRED_NUMBER;
   points: DocumentField<Array<[x: number, y: number]>> & {
-    type: [typeof Array];
+    type: typeof Array[];
```

Comment:
I think we should leave it as it was unless you have a specific reason for changing it?